odl/util/utility.py

```diff
@@ -417,6 +417,36 @@ def normalized_index_expression(indices, shape, int_to_slice=False):
     return tuple(indices)
 
 
+def snr(signal, noise, impl='general'):
+    """Compute the signal-to-noise ratio.
+    This compute::
+        impl='general'
+            SNR = s_power / n_power
+        impl='dB'
+            SNR = 10 * log10 (
+                s_power / n_power)
+    Parameters
+    ----------
+    signal : projection
+    noise : white noise
+    impl : implementation method
+    """
+    if np.abs(np.asarray(noise)).sum() != 0:
+        ave1 = np.sum(signal)/signal.size
```

Comment:
Use built in functions where possible. Here [np.std](http://docs.scipy.org/doc/numpy-1.10.0/reference/generated/numpy.std.html):

```python
s_power = np.std(signal)
n_power = np.std(noise)
```